# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/attributes/test_obj_group.html
# Commit: c0a7fdc4bfca
# Full Hash: c0a7fdc4bfca0596ffecd347f0e248440e371989
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P5: Unify GetLevelInternal and GetDefaultLevel as Accessible::GetLevel. r=Jamie
#   
#   The nsAccUtils method was a "fast" one for calculating set sizes and
#   conceptual parents. Unified it with
#   LocalAccessible::GetLevelInternal in Accessible::GetLevel with an
# ==============================================================================

diff -r f6ce0e05c92e -r c0a7fdc4bfca accessible/tests/mochitest/attributes/test_obj_group.html
--- a/accessible/tests/mochitest/attributes/test_obj_group.html	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/tests/mochitest/attributes/test_obj_group.html	Tue Jan 04 21:01:36 2022 +0000
@@ -32,13 +32,11 @@
       testGroupAttrs("opt2", 2, 2);
 
       // ////////////////////////////////////////////////////////////////////////
-      // HTML select with options
-      // XXX bug 469123
-      // testGroupAttrs("select2_optgroup", 1, 3, 1);
-      // testGroupAttrs("select2_opt3", 2, 3, 1);
-      // testGroupAttrs("select2_opt4", 3, 3, 1);
-      // testGroupAttrs("select2_opt1", 1, 2, 2);
-      // testGroupAttrs("select2_opt2", 2, 2, 2);
+      // HTML select with optgroup
+      testGroupAttrs("select2_opt3", 1, 2, 1);
+      testGroupAttrs("select2_opt4", 2, 2, 1);
+      testGroupAttrs("select2_opt1", 1, 2, 2);
+      testGroupAttrs("select2_opt2", 2, 2, 2);
 
       // ////////////////////////////////////////////////////////////////////////
       // HTML input@type="radio" within form